### ISSUE TITLE
Shipping methods not show at cart page

### ISSUE THREAD

**Issue by mikailomaki** [woocommerce/woocommerce#23531]
**Describe the bug**
A clear and concise description of what the bug is. Please be as descriptive as possible; issues lacking detail, or for any other reason than to report a bug, may be closed without action.

**To Reproduce**

**Screenshots**

**Expected behavior**

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by mikailomaki** [comment]
I updated my customer's WooCommerce to 3.6.2 this morning and noticed that shipping options are not show at cart page. It only says that "write the address to get shipping options". The same issue was on WC 3.5.0 and it worked when I updated 3.5.1. So is there the same bug again or what to do to make shipping options to show on cart page?

**Comment by claudiosanches** [comment]
This messages sounds fine, you really need to fill the address before get shipping options, also this may be related with geo location, if it isn't installed, there's no way to know where the customer is, so it's required to fill the address.

**Comment by mikailomaki** [comment]
I think you didn't understand the question exactly. Shipping options and costs are the same, no matter where customer is. The shop delivers only from Finland to Finland. And it has worked fine ftom 3.5.1 to 3.6.1 so in my opionion there is the bug. At some other forum someone said to someone with same issue that there is bug on 3.5.0, you hould use 3.5.1 We had same problem then and when we updated 3.5.1 it has been work since this day.

So I think issue should be re-opened.

**Comment by claudiosanches** [comment]
@mikailomaki I understand now what you are reporting, sorry!
But we already have a patch for it https://github.com/woocommerce/woocommerce/pull/23528
This will be included into 3.6.3.
